src/claude-session-utility.ts

```
@@ -176,24 +192,51 @@ export const readClaudeSessions = (limit = 100): ClaudeSession[] => {
+  snippets: Record<string, SessionSearchMatch>;
+}
+
 /**
- * Search Claude Code sessions by project name or first message
+ * Full search across ALL sessions (not just the ~100 the UI loads) and ALL
+ * user prompts (not just first/last) — fixes issue #131. Sessions come back
+ * newest-first; prompt text stays in this process (only snippets cross IPC).
  */
-export const searchClaudeSessions = (query: string, limit = 50): ClaudeSession[] => {
-  const allSessions = readClaudeSessions(500);
+export const searchClaudeSessions = (query: string, limit = 100): SessionSearchResult => {
```

ESLint

[error] Replace `query:·string,·limit·=·100` with `⏎··query:·string,⏎··limit·=·100,⏎`

(prettier/prettier)